scripts/xpp_update_happi_line.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python
+"""
+Updates items in the happi database to reflect the which line XPP is 
+using (mono line, pink line).
+
+Assumes pcds-conda, and grabs the currently active happi config.
+"""
+
+import argparse
+
+import click
+import happi
+from ophyd import EpicsSignalRO
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        prog='xpp_update_happi_line', description=__doc__
+    )
+
+    group = parser.add_mutually_exclusive_group()
```

Comment:
I wish I knew about `add_mutually_exclusive_group` before some CLI stuff I did recently

Reply:
Thanks to @vespos for this one